glossary.asciidoc

```diff
@@ -488,13 +483,13 @@ transaction::
     Transactions can also contain contracts and should not be confused with payments.
 
 transaction malleability::
-    Transaction malleability is a property that hash of a transaction can change without changing the semantic of the transaction (the UTXOs it is spending, the destinations and the corresponding amounts).
+    Transaction malleability is a property that the hash of a transaction can change without changing the semantic of the transaction (the UTXOs it is spending, the destinations and the corresponding amounts).
     For example, altering the signature can change the hash of a transaction, because of the non-deterministism of ECDSA signing.
-    A commitment transaction needs the hash of a funding transaction, if the hash of the funding transaction changes, transactions depending on it will become invalid. This will make users unable to claim the refunds if there is.
-    Segregated Witness soft fork addresses this issue and therefore is an important upgrade to support Lightning Network.
+    A commitment transaction needs the hash of a funding transaction and if the hash of the funding transaction changes, transactions depending on it will become invalid. This will make users unable to claim the refunds if there are any.
+    The segregated Witness soft fork addresses this issue and is therefore an important upgrade to support Lightning Network.
 
 transport layer::
-    In computer networking, the transport layer is a conceptual division of methods of a model of how computers talk to each other.
+    In computer networking, the transport layer is a conceptual model of how computers talk to each other.
```

Comment:
I'm not sure I love this cut

Reply:
The problem is the concatenation of "divisions", "methods" and "model".  How can you have "methods of a model"?  I think the original wording is partially taken from Wikipedia.

How about: 
In computer networking, the transport layer is a conceptual division of the methods used by computers and applications to talk to each other.

Reply:
Agreed, I like the sentence :)
By the way, I think it was an adaptation of the wikipedia page: https://en.wikipedia.org/wiki/Transport_layer